charts/controller/templates/controller-deployment.yaml

```diff
@@ -125,6 +125,10 @@ spec:
                   key: password
             - name: RESERVED_NAMES
               value: "deis, deis-builder, deis-workflow-manager, grafana"
+            - name: WORKFLOW_NAMESPACE
```

Comment:
How does the environmental variable translate into the YAML file here? Is this a convention that we replace out later or something?